deploy/README.md

```diff
@@ -287,15 +294,20 @@ FOREIGN_RPC_URL=https://mainnet.infura.io
 # Address on Foreign network with permissions to change parameters of the bridge contract.
 # For extra security we recommended using a multi-sig wallet contract address here.
 FOREIGN_BRIDGE_OWNER=0x
-# Address on Foreign network with permissions to change parameters of bridge validator contract.
+# Address on the Foreign network with permissions to change parameters of 
+# the bridge validator contract.
 FOREIGN_VALIDATORS_OWNER=0x
-# Address on Foreign network with permissions to upgrade the bridge contract and the bridge validator contract.
+# Address on the Foreign network with permissions to upgrade the bridge contract 
+# and the bridge validator contract.
 FOREIGN_UPGRADEABLE_ADMIN=0x
-# These three parameters are not used in this mode, but the deployment script
-# requires it to be set to some value.
-FOREIGN_DAILY_LIMIT=0
-FOREIGN_MAX_AMOUNT_PER_TX=0
-FOREIGN_MIN_AMOUNT_PER_TX=0
+# The daily transaction limit in Wei. This is used on the Home side to check 
+# the bridge validator’s actions.
+FOREIGN_DAILY_LIMIT=15000000000000000000000000
+# The maximum limit for one transaction in Wei. FOREIGN_MAX_AMOUNT_PER_TX must be 
```

Comment:
Do you think it is worth to specify `This is used on the Home side to check the bridge validator’s actions.` here as well?

Reply:
Yes, I will add.